2024-02-25 21:47:40,881 NanoPlot 1.42.0 started with arguments Namespace(N50=True, alength=False, bam=None, barcoded=False, color='#4CB391', colormap='Greens', cram=None, downsample=None, dpi=100, drop_outliers=False, fasta=None, fastq=['../0_raw/SRR11673974_1.fastq.gz'], fastq_minimal=None, fastq_rich=None, feather=None, font_scale=1, format=['png'], hide_stats=False, huge=False, info_in_report=False, legacy=['dot'], listcolormaps=False, listcolors=False, loglength=False, maxlength=None, minlength=None, minqual=None, no_N50=False, no_static=False, no_supplementary=False, only_report=False, outdir='Hjor_nanoplot', path='Hjor_nanoplot/', percentqual=False, pickle=None, plots=['dot'], prefix='', raw=False, readtype='1D', runtime_until=None, store=False, summary=None, threads=10, title=None, tsv_stats=False, ubam=None, verbose=False)
2024-02-25 21:47:40,882 Python version is: 3.8.18 | packaged by conda-forge | (default, Dec 23 2023, 17:21:28)  [GCC 12.3.0]
2024-02-25 21:47:41,088 Nanoget: Starting to collect statistics from plain fastq file.
2024-02-25 21:47:41,090 Nanoget: Decompressing gzipped fastq ../0_raw/SRR11673974_1.fastq.gz
2024-02-25 22:14:03,181 Reduced DataFrame memory usage from 63.7298583984375Mb to 63.7298583984375Mb
2024-02-25 22:14:03,508 Nanoget: Gathered all metrics of 4176592 reads
2024-02-25 22:14:06,418 Calculated statistics
2024-02-25 22:14:06,418 Using sequenced read lengths for plotting.
2024-02-25 22:14:06,601 NanoPlot:  Valid color #4CB391.
2024-02-25 22:14:06,603 NanoPlot:  Valid colormap Greens.
2024-02-25 22:14:07,164 NanoPlot:  Creating length plots for Read length.
2024-02-25 22:14:07,169 NanoPlot: Using 4176592 reads with read length N50 of 7346bp and maximum of 714786bp.
2024-02-25 22:14:14,101 Saved Hjor_nanoplot/WeightedHistogramReadlength  as png (or png for --legacy)
2024-02-25 22:14:15,853 Saved Hjor_nanoplot/WeightedLogTransformed_HistogramReadlength  as png (or png for --legacy)
2024-02-25 22:14:17,462 Saved Hjor_nanoplot/Non_weightedHistogramReadlength  as png (or png for --legacy)
2024-02-25 22:14:19,035 Saved Hjor_nanoplot/Non_weightedLogTransformed_HistogramReadlength  as png (or png for --legacy)
2024-02-25 22:14:28,251 Saved Hjor_nanoplot/Yield_By_Length  as png (or png for --legacy)
2024-02-25 22:14:28,260 Created length plots
2024-02-25 22:14:28,354 NanoPlot: Creating Read lengths vs Average read quality plots using 4176592 reads.
2024-02-25 22:14:31,159 Saved Hjor_nanoplot/LengthvsQualityScatterPlot_dot  as png (or png for --legacy)
2024-02-25 22:14:34,577 NanoPlot: Creating Read lengths vs Average read quality legacy plots using 10000 reads.
2024-02-25 22:14:34,681 findfont: Found a missing font file.  Rebuilding cache.
2024-02-25 22:14:37,878 Failed to extract font properties from /usr/share/fonts/google-noto-emoji/NotoColorEmoji.ttf: In FT2Font: Can not load face.  Unknown file format.
2024-02-25 22:14:37,912 generated new fontManager
2024-02-25 22:14:38,025 'PathCollection' object has no property 'stat_func'
Traceback (most recent call last):
  File "/home/felipe.peres/.conda/envs/metagenomics/lib/python3.8/site-packages/nanoplot/NanoPlot.py", line 110, in main
    plots = make_plots(datadf, settings)
  File "/home/felipe.peres/.conda/envs/metagenomics/lib/python3.8/site-packages/nanoplot/NanoPlot.py", line 179, in make_plots
    nanoplotter.scatter(
  File "/home/felipe.peres/.conda/envs/metagenomics/lib/python3.8/site-packages/nanoplotter/nanoplotter_main.py", line 182, in scatter
    plots_made += scatter_legacy(
  File "/home/felipe.peres/.conda/envs/metagenomics/lib/python3.8/site-packages/nanoplotter/nanoplotter_main.py", line 295, in scatter_legacy
    plot = sns.jointplot(
  File "/home/felipe.peres/.local/lib/python3.8/site-packages/seaborn/axisgrid.py", line 2255, in jointplot
    grid.plot_joint(scatterplot, **joint_kws)
  File "/home/felipe.peres/.local/lib/python3.8/site-packages/seaborn/axisgrid.py", line 1826, in plot_joint
    func(x=self.x, y=self.y, **kwargs)
  File "/home/felipe.peres/.local/lib/python3.8/site-packages/seaborn/relational.py", line 761, in scatterplot
    p.plot(ax, kwargs)
  File "/home/felipe.peres/.local/lib/python3.8/site-packages/seaborn/relational.py", line 573, in plot
    points = ax.scatter(x=x, y=y, **kws)
  File "/home/felipe.peres/.local/lib/python3.8/site-packages/matplotlib/__init__.py", line 1361, in inner
    return func(ax, *map(sanitize_sequence, args), **kwargs)
  File "/home/felipe.peres/.local/lib/python3.8/site-packages/matplotlib/axes/_axes.py", line 4597, in scatter
    collection.update(kwargs)
  File "/home/felipe.peres/.local/lib/python3.8/site-packages/matplotlib/artist.py", line 1062, in update
    raise AttributeError(f"{type(self).__name__!r} object "
AttributeError: 'PathCollection' object has no property 'stat_func'
